run_cbmc.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/zsh
+
+CBMC_DIR=$HOME/Development/cbmc/build/bin/
```

Comment:
```suggestion
: ${CBMC_DIR:="$HOME/Development/cbmc/build/bin/"}
```

That way someone can override it with their own environment variable setting.